CHANGELOG.md

```diff
@@ -1,146 +0,0 @@
-# Change Log
```

Comment:
Also saw we have this outdated changelog. Do we want to migrate this into `master` and refresh it a bit?